modules/rostests/apitests/ntdll/NtAllocateVirtualMemory.c

```diff
@@ -561,8 +560,37 @@ CheckAdjacentVADs()
     BaseAddress = UlongToPtr(0x50010000);
     Size = 0x10000;
     Status = NtFreeVirtualMemory(NtCurrentProcess(), &BaseAddress, &Size, MEM_RELEASE);
+    // Result varies between ReactOS and Windows.
     ok_ntstatus(Status, STATUS_SUCCESS);
-
+    if (!NT_SUCCESS(Status))
+    { // Unexpected, cleanup.
```

Comment:
Comment on its own line please.
And use /* */ comments like the rest of the file